README.md

```diff
@@ -67,86 +84,58 @@ The table below outlines what version of Nexus Repository the plugin was built a
+
    ```
          <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
-   +     <feature version="1.0.3" prerequisite="false" dependency="false">nexus-repository-r</feature>
+   +     <feature version="1.1.1.SNAPSHOT" prerequisite="false" dependency="false">nexus-repository-r</feature>
          <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-yum</feature>
      </feature>
    ```
    And
    ```
-   + <feature name="nexus-repository-r" description="org.sonatype.nexus.plugins:nexus-repository-r" version="1.0.3">
+   + <feature name="nexus-repository-r" description="org.sonatype.nexus.plugins:nexus-repository-r" version="1.1.1.SNAPSHOT">
    +     <details>org.sonatype.nexus.plugins:nexus-repository-r</details>
-   +     <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-r/1.0.3</bundle>
+   +     <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-r/1.1.1-SNAPSHOT</bundle>
    + </feature>
     </features>
    ```
 This will cause the plugin to be loaded and started with each startup of Nexus Repository.
 
-## The Fine Print
+### Easiest Install for version  
+Relevant only for Nexus Repository manager version 3.18.0-01 and lower  which doest includes `R` plugin.
+But we strongly recommend to update to the latest Nexus Repository version.
 
-It is worth noting that this is **NOT SUPPORTED** by Sonatype, and is a contribution of ours
-to the open source community (read: you!)
+Thanks to some upstream work in Nexus Repository (versions newer than 3.15), it's become a LOT easier to install a plugin. To install the `R` plugin, you can either build locally or download from The Central Repository:
+
+#### Option 1: Build a *.kar file locally from the GitHub Repo
+* Clone this repo and `cd` to the cloned directory location
+* Build the plugin with `mvn clean package -PbuildKar`
+* There should now be a `nexus-repository-r-1.0.4-bundle.kar` file your `<cloned_repo>/target` directory 
 
-Remember:
+#### Option 2: Download a *.kar file from The Central Repository 
+* Download `nexus-repository-r-1.0.4-bundle.kar` from [The Central Repository](https://search.maven.org/artifact/org.sonatype.nexus.plugins/nexus-repository-r/1.0.4/bundle)
 
-* Use this contribution at the risk tolerance that you have
-* Do NOT file Sonatype support tickets related to R support
-* DO file issues here on GitHub, so that the community can pitch in
+Once you've completed Option 1 or 2, copy the `nexus-repository-r-1.0.4-bundle.kar` file into the `<nexus_dir>/deploy` folder for your Nexus Repository installation.
+
+Restart Nexus Repo, or go ahead and start it if it wasn't running to begin with.
+
+You should see the R repository types (e.g. `r (proxy)`) in the available Repository Recipes to use, if all has gone according to plan :)
+
+## The Fine Print
+
+Starting from version 3.20.0 `R` plugin is supported by Sonatype, but still is a contribution of ours
```

Comment:
Suggestion: "**the** `R` plugin".

Reply:
2218cd10cffeabcec77dd56d1c3837388b45f098